code/controllers/subsystems/necromorph.dm

```diff
@@ -6,6 +6,7 @@ SUBSYSTEM_DEF(necromorph)
 	//Necromorph Lists
 	var/list/major_vessels = list()	//Necromorphs that need a player to control them. they are inert husks without one.
 	var/list/minor_vessels	=	list()	//Necromorphs that have AI and don't need a player, but can be possessed anyway if someone wants to do manual control
+	var/list/all_necros		= list()	//List of all alive necromorphs
```

Comment:
redundant, this is what major vessels is for

Reply:
I thought major vessels are for those necros that don't have player yet